sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosConstants.java

```diff
@@ -11,6 +11,7 @@
     @SuppressWarnings("PMD.AvoidUsingHardCodedIP")
     public static final String MESOS_MASTER_ZK_CONNECTION_STRING = MASTER_MESOS + ":2181";
     public static final String MESOS_MASTER_URI = "http://" + MASTER_MESOS;
+    public static final Boolean DEFAULT_GPU_POLICY = true;
```

Comment:
@klueska:  Is it recommended according to your original design for scarce resources that all frameworks opt-in by default?